src/net.cpp

```diff
@@ -2185,18 +2305,33 @@ uint64_t CNode::GetOutboundTargetBytesLeft()
     return (nMaxOutboundTotalBytesSentInCycle >= nMaxOutboundLimit) ? 0 : nMaxOutboundLimit - nMaxOutboundTotalBytesSentInCycle;
 }
 
-uint64_t CNode::GetTotalBytesRecv()
+uint64_t CConnman::GetTotalBytesRecv()
 {
     LOCK(cs_totalBytesRecv);
     return nTotalBytesRecv;
 }
 
-uint64_t CNode::GetTotalBytesSent()
+uint64_t CConnman::GetTotalBytesSent()
 {
     LOCK(cs_totalBytesSent);
     return nTotalBytesSent;
 }
 
+uint64_t CConnman::GetLocalServices() const
+{
+    return nLocalServices;
+}
+
+void CConnman::SetBestHeight(int height)
```

Comment:
nit: I think this could use documentation that the setter requires cs_main to be held to prevent older values from overwriting newer values. Also, I don't see any other data that would need to be synchronized with the height via acquire/release semantics; if there are no writes before the release that need to be visible side-effects after the acquire, relaxed load/store would be clearer.